examples/kubernetes-cluster/main.tf

```diff
@@ -55,9 +55,17 @@ module "master" {
     data_path          = "/var/lib/etcd"
   }
 
+  override_binaries = {
```

Comment:
I assume it can use the default value in the terraform module [terraform-ignition-kubernetes](https://github.com/getamis/terraform-ignition-kubernetes/pull/60/files#diff-ee80d9f1b8c52d7c16cc9f81bc814e37d9342ffa50eb3aa9f8a83dbe91a40a95R8-R11), so it isn't necessary to declare override_binaries's value here